requirements.txt

```diff
@@ -11,3 +11,4 @@ requests
 rich
 setuptools
 termcolor
+pytest-html
```

Comment:
hm, I don't think we should add this plugin by default because anyone who is not doing development won't need it. this dependency can be added only for the github workflow, or can be manually installed by any developer.